.travis.yml

```diff
@@ -18,6 +18,10 @@ matrix:
   - php: 7.1
     env:
     - DEPS=lowest
+  - php: 7.1
+    env:
+    - EXECUTE_TEST_PHPSTAN=true
```

Comment:
We should execute it on locked dependencies

Reply:
composer.lock is not committed...

Reply:
Sorry, I meant we should not use a duplicated environment.